.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6178416)